webpack.config.js

```diff
@@ -139,7 +112,8 @@ module.exports = {
       title: 'Front-End Bootstrap',
       template: './src/views/index/index.pug',
       filename: 'index.html'
-    })
+    }),
+    new ESLintPlugin(eslintOptions)
```

Comment:
I think it's cleaner to have the options in here since you are not going to reuse them 😁